Multisig/UI/Settings/OwnerKeyManagement/OwnerKeysListViewController/OwnerKeysListTableViewCell.swift

```diff
@@ -16,7 +16,7 @@ class OwnerKeysListTableViewCell: UITableViewCell {
 
         addressInfoView.setDetailImage(nil)
         addressInfoView.copyEnabled = false
-        addressInfoView.setDetailImage(UIImage(named: "arrow"))
+        addressInfoView.setDetailImage(nil)
```

Comment:
@MouazAlzahabi there's already such a line in line 17 - do you need this one?

Reply:
no, duplication by mistake